usr/share/rear/lib/_input-output-functions.sh

```diff
@@ -791,6 +791,34 @@ preferably the whole debug information via 'rear -D $WORKFLOW'
 ===================="
 }
 
+# Error out with a deprecation info
+# first arg is feature keyword
+# everything else is a reason for the deprecation
+function ErrorIfDeprecated () {
+    local feature="$1" ; shift
+
+    if IsInArray "$feature" "${DISABLE_DEPRECATION_ERRORS[@]}" ; then
+        LogPrint "Disabled deprecation error for >$feature<"
```

Comment:
Nitpicking (feel free to ignore):
Usually we use `'...'` in messages to indicate fixed terms
but I see some code places where also `<<...>>` or similar is used.
Personally I prefer `'...'` because that are usual quoting characters
(but they cannot be used for nested quotings).

Reply:
Yes, personally I like this `>...<` format as it won't conflict with anything but I can switch to single quotes if you prefer.

Reply:
Use what you prefer.
I don't want strict style rules.
Different styles in ReaR show the user that
ReaR is a living project where various people contribute.

A question out of curiosity:

Personally I prefer `'...'` within `"..."` like
```
"some normal text 'fixed term' more normal text"
```
because from my experience that makes the code easiest to read
so I would like to understand how the `>...<` format is better?

My reasoning:

For me `>...<` could make the code look confusing because
normally `<` and `>` are used for redirections in bash
so when `<` and `>` are also used for quoting fixed terms
inside strings then complicated code lines like
```
build/default/990_verify_rootfs.sh:
PrintError "$( grep 'not found' <<<"$ldd_output" )"
```
become even more hard to understand when in
messages `<` and `>` are used for redirections
and also used to quote fixed terms like
(an artificial example)
```
PrintError "Error in >$executable<: >$( grep 'not found' <<<"$ldd_output" )<"
```
that on command line:
```
# executable=/path/to/prog

# ldd_output="$( echo -e "foo\nbar not found therein\nbaz" )"

# echo "Error in >$executable<: >$( grep 'not found' <<<"$ldd_output" )<"
Error in >/path/to/prog<: >bar not found therein<
```
versus my personally preferred style
```
# echo "Error in '$executable': '$( grep 'not found' <<<"$ldd_output" )'"
Error in '/path/to/prog': 'bar not found therein'
```
where I personally find the code easier to read
regardless that the `'` in the `grep 'not found'` part
have a bit different meaning than the other `'`
but all have same kind of meaning: quoting.

I also find the output message easier to read
because `'` in the output is a usual quoting character
but for me it is primarily about
"Code must be easy to read" in
https://github.com/rear/rear/wiki/Coding-Style

Reply:
you convinced me here